YAPA/MainWindow.xaml.cs

```diff
@@ -100,29 +99,21 @@ public MainWindow()
             BreakTrayIconColor = (System.Drawing.Color)System.Drawing.ColorTranslator.FromHtml(Properties.Settings.Default.BreakTrayIconColor);
         }
 
-        private void _musicPlayer_MediaEnded1(object sender, EventArgs e)
-        {
-            if (_musicPlayer.Source != null && File.Exists(_musicPlayer.Source.AbsolutePath))
-            {
-                _musicPlayer.Play();
-            }
-        }
-
         private void PlayMusic()
         {
             if (_isWork)
             {
                 if (File.Exists(WorkMusic))
                 {
-                    _musicPlayer.Open(new Uri(WorkMusic));
-                    _musicPlayer.Play();
+                    _musicPlayer.SoundLocation = WorkMusic;
+                    _musicPlayer.PlayLooping();
                 }
             }
             else
             {
                 if (File.Exists(BreakMusic))
                 {
-                    _musicPlayer.Open(new Uri(BreakMusic));
+                    _musicPlayer.SoundLocation = BreakMusic;
                     _musicPlayer.Play();
```

Comment:
you're right, but what I thought is maybe breakMusic does not need to loop, because it's a break time, user probably need a real break, he may not need something looping around his ear.